tests/default/indices/msearch.yaml

```diff
@@ -0,0 +1,124 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+
+description: Test endpoints relevant to the lifecycle of an index, including multi-get and multi-search operations.
+epilogues:
+  - path: /books
+    method: DELETE
+    status: [200, 404]
+  - path: /games
+    method: DELETE
+    status: [200, 404]
+chapters:
+  - synopsis: Create an index named `books` with mappings and settings.
+    path: /{index}
+    method: PUT
+    parameters:
+      index: books
+    request:
+      payload:
+        mappings:
+          properties:
+            name:
+              type: keyword
+            age:
+              type: integer
+        settings:
+          number_of_shards: 5
+          number_of_replicas: 2
+    response:
+      status: 200
+
+  - synopsis: Create an index named `games` with default settings.
+    path: /{index}
+    method: PUT
+    parameters:
+      index: games
+
+  - synopsis: Add documents to the `books` index.
+    path: /{index}/_doc/{id}
+    method: POST
+    parameters:
+      index: books
+      id: '1'
+    request:
+      payload:
+        name: Book 1
+        age: 10
+    response:
+      status: 201
+
+  - synopsis: Add more documents to the `books` index.
+    path: /{index}/_doc/{id}
+    method: POST
+    parameters:
+      index: books
+      id: '2'
+    request:
+      payload:
+        name: Book 2
+        age: 20
+    response:
+      status: 201
+
+  - synopsis: Add another document to the `books` index.
+    path: /{index}/_doc/{id}
+    method: POST
+    parameters:
+      index: books
+      id: '3'
+    request:
+      payload:
+        name: Book 3
+        age: 30
+    response:
+      status: 201
```

Comment:
These should be in a prologue. You can also use `_bulk` from other examples which makes it much shorter.